apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -906,6 +906,12 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
         Blockly.blockly_.ShortcutRegistry.registry.unregister('redo');
         blocklyWrapper.KeyboardNavigation.dispose();
       }
+      // Add the keyboard shortcuts div. This is here to ensure it happens
+      // prior to keyboard nav installation
+      const shortcutDialog = document.createElement('div');
+      shortcutDialog.id = 'shortcuts';
+      document.body.appendChild(shortcutDialog);
```

Comment:
Where this is currently positioned, I think we'll create the div even we've already initialized keyboard nav (on a previous lab2 level). Does this potentially create additional divs with the same id? I wonder if we should consider only doing this the first time (ie. if `!blocklyWrapper.KeyboardNavigation`) or actually just querying for the element to ensure it doesn't exist before creating it.

Reply:
https://github.com/code-dot-org/code-dot-org/pull/66183 does this work?